packages/web3-wrapper/src/web3_wrapper.ts

```diff
@@ -149,12 +150,17 @@ export class Web3Wrapper {
      * @param   callAndTxnDefaults  Override Call and Txn Data defaults sent with RPC requests to the backing Ethereum node.
      * @return  An instance of the Web3Wrapper class.
      */
-    constructor(supportedProvider: SupportedProvider, callAndTxnDefaults: Partial<CallData> = {}) {
+    constructor(
+        supportedProvider: SupportedProvider,
+        callAndTxnDefaults: Partial<CallData> = {},
+        jsonRpcIdNameSpace?: string,
+    ) {
         this.abiDecoder = new AbiDecoder([]);
         this._supportedProvider = supportedProvider;
         this._provider = providerUtils.standardizeOrThrow(supportedProvider);
         this._callAndTxnDefaults = callAndTxnDefaults;
         this._jsonRpcRequestId = 1;
+        this._jsonRpcRequestNameSpace = jsonRpcIdNameSpace;
```

Comment:
nit: if there's no reason to change the name I think it's better to keep it consistent as `jsonRpcIdNameSpace` throughout